crates/sui-core/src/consensus_adapter.rs

```diff
@@ -418,9 +418,9 @@ impl ConsensusAdapter {
         let (position, positions_moved, preceding_disconnected) =
             self.submission_position(committee, tx_digest);
 
-        const DEFAULT_LATENCY: Duration = Duration::from_secs(3); // > p50 consensus latency with global deployment
+        const DEFAULT_LATENCY: Duration = Duration::from_secs(1); // > p50 consensus latency with global deployment
```

Comment:
What do you guys think about being more aggressive here? 750ms or even maybe 600ms

Reply:
We can reduce the value as we increase the round rate cap.